scripts/test-export-ciphers-rejected.py

```diff
@@ -33,30 +32,36 @@ def help_msg():
     print("                names and not all of them, e.g \"sanity\"")
     print(" -e probe-name  exclude the probe from the list of the ones run")
     print("                may be specified multiple times")
-    print(" --ssl3         expect SSLv3 to be enabled")
+    print(" -n num         only run `num` random tests instead of a full set")
+    print("                (excluding \"sanity\" tests)")
+    print(" --min-ver val  The lowest version support, \"SSLv3\" by default")
+    print("                may be \"TLSv1.0\", \"TLSv1.1\" or \"TLSv1.2\"")
     print(" --help         this message")
 
 
 def main():
     host = "localhost"
     port = 4433
+    num_limit = None
     run_exclude = set()
-    ssl3 = False
+    min_ver = (3, 0)
```

Comment:
given that most implementations are getting rid of 3.0, shouldn't this be 3.1 by default?

Reply:
no, the scripts should test as much as possible and any exceptions should be a conscious decision by the user preparing the json script; if the server does support SSLv3 and the SSLv3 was excluded (as it was previously), then SSLv3 may unintentionally remain untested